lib/fx/adapters/postgres.rb

```diff
@@ -125,7 +125,11 @@ def update_trigger(name, on:, sql_definition:)
       #
       # @return [void]
       def drop_function(name)
-        execute "DROP FUNCTION #{name}();"
+        fns = functions.select { |function| function.name == name.to_s }
```

Comment:
Great suggestion. I think my original implementation had a `functions_by_name` further up the tree that I didn't like, but a helper method locally here makes a ton of sense.